extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -843,7 +854,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         // the size header for fluid ads. Fluid size headers always come back as
         // 0x0.
         !(size.width == 0 && size.height == 0);
-    setStyles(dev().assertElement(this.iframe), {
+    setStyles(dev().assert(this.iframe), {
```

Comment:
Why this change? `assertElement` calls `assert`.

Reply:
Nevermind, just saw your recent commit.

Reply:
I reverted... had an issue where I was attempting to make this.iframe as an object instead of element causing this to fail.